### PR TITLE
zb: populate images first for the GetCatalog test

### PR DESCRIPTION
**Pull request by roxanaN**
Signed-off-by: Roxana Nemulescu <roxana.nemulescu@gmail.com>

**Which issue does this PR fix**:
GetCatalog() uses empty storage for benchmarking.

**What does this PR do / Why do we need it**:
Populate with mix of many small, medium and large images.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — APPROVED** [review]
LGTM.

File: cmd/zb/perf.go [hunk 1]
```diff
@@ -307,13 +307,38 @@ type testFunc func(
...
+	sizes := [3]int{
+		smallBlob,
+		mediumBlob,
+		largeBlob,
+	}
+
+	mrand.Seed(time.Now().UTC().UnixNano())
+	imagesNo := mrand.Intn(requests) + 1
+
+	for imageNo := 0; imageNo < imagesNo; imageNo++ {
```
Comment (rchincha) [inline cmd/zb/perf.go:331]: This is unpredictable, so how do I now compare against other registries or across runs.
Would say, make it 70-20-10 (small-medium-large) split.

**Review by andaaron — APPROVED** [review]
LGTM.

Although I'm not convinced that, in principle, the testConfig object should be changed inside the test function itself, but I leave that to you guys to decide.

**Comment by rchincha** [comment]
```
============
Test name:            Get Catalog
Time taken for tests: 6.839335311s
Complete requests:    100
Failed requests:      0
Requests per second:  14.621304

1MB:   74
10MB:  14
100MB: 12

2xx responses: 100

min: 8.171493ms
max: 160.435317ms
p50: 22.821459ms
p75: 39.814951ms
p90: 65.624667ms
p99: 160.435317ms
```

**Comment by rchincha** [comment]
https://github.com/project-zot/zot/blob/main/.github/workflows/benchmark.yaml#L29
"          key: ${{ runner.os }}-benchmark"

Call this "$${runner.os }}-gen1-benchmark" so that we don't get a cache hit and try.

**Review by peusebiu — APPROVED** [review]
LGTM

**Review by rchincha — DISMISSED** [review]
lgtm

**Review by andaaron — APPROVED** [review]

**Pull request merged**